src/nifake/unit_tests/test_library_singleton.py

```diff
@@ -0,0 +1,7 @@
+import nifake
+import pytest
+
+
+def test_get_uninstalled_driver_library_raises_driver_not_installed_error():
```

Comment:
Let me know if you prefer `runtime` over `library` in this test name.

Reply:
how about: `test_driver_runtime_not_installed_raises_driver_not_installed_error`